### PR TITLE
Miscellaneous fixes and improvements for WFL

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- No ticket is linked to this PR.

### Changes

- Upgrade the version of Arrays pipeline to `v1.9.1`. (see https://broadinstitute.slack.com/archives/CESEYJW9W/p1596465923385200)
- Use tempdir to avoid polluting file system when running cloud function tests locally. 
- Refactor the specs so they are grouped in a reasonable way for future development.
- Remove a wgs workflow test from the endpoint testing, remove that endpoint as well.
- Switch to use the `documentation` reporter as the default progress of kaocha. 

    Before:
    ```
    unit:   100% [==================================================] 13/13
    13 tests, 60 assertions, 0 failures.
    ```

    After:
    ```
    --- integration (clojure.test) ---------------------------
    zero.integration.v1-endpoint-test
      test-exec-wgs-workload
        The `exec` endpoint creates and starts a WGS workload
      test-append-to-aou-workload
        The `append_to_aou` endpoint appends a new workflow to aou workload.
      test-create-aou-workload
        The `create` endpoint creates new AOU workload
      test-create-wgs-workload
        The `create` endpoint creates new WGS workload
      test-start-aou-workload
        The `start` endpoint starts an existing AOU workload
      test-exec-aou-workload
        The `exec` endpoint creates and starts an AOU workload
      test-copyfile-workload
      test-start-wgs-workload
        The `start` endpoint starts an existing WGS workload

    8 tests, 31 assertions, 0 failures.

    Top 1 slowest kaocha.type/clojure.test (233.39679 seconds, 100.0% of total time)
      integration
        233.39679 seconds average (233.39679 seconds / 1 tests)

    Top 1 slowest kaocha.type/ns (233.38548 seconds, 100.0% of total time)
      zero.integration.v1-endpoint-test
        29.17319 seconds average (233.38548 seconds / 8 tests)

    Top 3 slowest kaocha.type/var (210.50641 seconds, 90.2% of total time)
      zero.integration.v1-endpoint-test/test-copyfile-workload
        198.12834 seconds zero/integration/v1_endpoint_test.clj:101
      zero.integration.v1-endpoint-test/test-create-wgs-workload
        6.31759 seconds zero/integration/v1_endpoint_test.clj:22
      zero.integration.v1-endpoint-test/test-exec-aou-workload
        6.06048 seconds zero/integration/v1_endpoint_test.clj:94
    ```

### Review Instructions

- No instructions.

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -66,11 +66,11 @@
...
-  {:extra-deps  {lambdaisland/kaocha {:mvn/version "1.0.632"}
-                 org.liquibase/liquibase-cdi {:mvn/version "4.0.0"
-                                              :exclusions [ch.qos.logback/logback-classic]}
-                 org.liquibase/liquibase-core {:mvn/version "4.0.0"
-                                               :exclusions [ch.qos.logback/logback-classic]}}
+  {:extra-deps  {lambdaisland/kaocha           {:mvn/version "1.0.632"}
+                 org.liquibase/liquibase-cdi   {:mvn/version "4.0.0"
+                                                :exclusions [ch.qos.logback/logback-classic]}
+                 org.liquibase/liquibase-core  {:mvn/version "4.0.0"
+                                                :exclusions [ch.qos.logback/logback-classic]}}
```
Comment (jack-r-warren) [inline deps.edn:73]: Did this come from `clojure -A:format`?

Comment (rexwangcc) [reply deps.edn:73]: I think this comes from cursive 🤔

File: src/zero/api/routes.clj [hunk 2]
```diff
@@ -52,15 +52,10 @@
             :parameters {:query {:environment string?}}
             :responses {200 {:body {:total pos-int?}}}
             :handler handlers/status-counts}}]
-   ["/api/v1/wgs"
-    {:post {:summary    "Submit WGS Reprocessing workflows"
-            :parameters {:body ::spec/wgs-request}
-            :responses  {200 {:body {:results vector?}}}
-            :handler    handlers/submit-wgs}}]
```
Comment (samanehsan) [inline src/zero/api/routes.clj:59]: Can you also remove the `submit-wgs` handler? I don't think anything else is using it

File: deps.edn (hunk 1 above)
Comment (tbl3rd) [reply deps.edn:73]: IWBNI all the tools used the same indentation.

File: src/zero/api/routes.clj (hunk 2 above)
Comment (rexwangcc) [reply src/zero/api/routes.clj:59]: Good catch! Removed.

File: src/zero/api/spec.clj [hunk 3]
```diff
@@ -1,103 +1,107 @@
...
 
-(s/def ::base_file_name       string?)
-(s/def ::commit               (s/and string? (comp (partial == 40) count)))
-(s/def ::created              inst?)
-(s/def ::creator              string?)
-(s/def ::cromwell             string?)
-(s/def ::end                  string?)
-(s/def ::environment          string?)
+;; shared
+(s/def ::environment string?)
```
Comment (samanehsan) [inline src/zero/api/spec.clj:11]: I like how you grouped the specs! I think alphabetizing them would be nice as well!

Comment (rexwangcc) [reply src/zero/api/spec.clj:11]: Nice suggestion! Let me alphabetize them within the current groups.

**Pull request merged**